moped-editor/src/utils/mapHelpers.js

```diff
@@ -128,9 +134,8 @@ export const mapConfig = {
     CTN: {
       layerLabel: "Streets",
       layerIdName: "ctn-lines",
-      layerIdField: "PROJECT_EXTENT_ID",
       tooltipTextProperty: "FULL_STREET_NAME",
-      layerIdGetPath: "properties.PROJECT_EXTENT_ID",
+      featureIdProperty: "PROJECT_EXTENT_ID",
```

Comment:
`layerID` was throwing me off, so I standardized around `featureId`. I think this term added to the the confusion that led us to have a redundant `layerIdField` prop.

Reply:
@johnclary The `layerIDGetPath` variable is meant to be used with Lodash, it's [basically a way to provide the path of an attribute in an object as a string for access](https://lodash.com/docs/4.17.15#get). 

The name of the variable is trivial but personally I'd appreciate having the word "path" to remind me what its purpose is and its relationship to Lodash, or at least a comment near the new name `featureIdProperty` that can help us in the future. 🙏 Regardless, it looks like the refactor is working.